packages/next/src/build/utils.ts

```diff
@@ -1818,6 +1780,54 @@ export async function isPageStatic({
     })
 }
 
+export function reduceAppConfig(
```

Comment:
I don't see this being outside of this file. Can we stop exporting if we don't need to, to avoid spreading specialized utils unnecessarily?

```suggestion
function reduceAppConfig(
```